fastlane/docs/Actions.md

```diff
@@ -876,8 +876,34 @@ crashlytics(
   ipa_path: './app.ipa'
 )
 ```
+
 Additionally you can specify `notes`, `emails`, `groups` and `notifications`.
 
+#### Distributing to Groups
+
+When using the `groups` parameter, it's important to use the group **alias** names for each group you'd like to distribute to. A group's alias can be found in the web UI. If you're viewing the Beta page, you can open the groups dialog here:
+
+![Crashlytics Beta Groups Navigation](../assets/Beta-Groups-Nav.png)
```

Comment:
Is it possible to put these images side by side using HTML? This adds quite a bit to the documentation here.